optimum/intel/neural_compressor/quantization.py

```diff
@@ -447,14 +449,10 @@ def _apply_quantization_from_config(q_config: Dict, model: torch.nn.Module) -> t
 class INCModel:
     TRANSFORMERS_AUTO_CLASS: ClassVar = AutoModel
 
-    def __init__(self, *args, **kwargs):
-        raise EnvironmentError(
-            f"{self.__class__.__name__} is designed to be instantiated using the"
-            f"`{self.__class__.__name__}.from_pretrained(model_name_or_path)` method."
-        )
-
     @classmethod
-    def from_pretrained(cls, model_name_or_path: str, q_model_name: Optional[str] = None, **kwargs) -> torch.nn.Module:
+    def from_pretrained(
+        cls, model_name_or_path: str, q_model_name: Optional[str] = None, export=False, **kwargs
```

Comment:
export will never be given no ? (not used)

```suggestion
        cls, model_name_or_path: str, q_model_name: Optional[str] = None, **kwargs
```

Reply:
I removed parent class "INCModel", because there are many issue if INCBaseModel inherit from the INCModel.